ibis/backends/tests/test_register.py

```diff
@@ -99,7 +99,8 @@ def gzip_csv(data_dir, tmp_path):
 )
 def test_register_csv(con, data_dir, fname, in_table_name, out_table_name):
     with pushd(data_dir / "csv"):
-        table = con.register(fname, table_name=in_table_name)
+        with pytest.warns(FutureWarning, match="v9.0"):
+            table = con.register(fname, table_name=in_table_name)
 
     assert any(out_table_name in t for t in con.list_tables())
     if con.name != "datafusion":
```

Comment:
I think this test should be rewritten to test `read_csv` (and we can probably rename the test file, too)

Reply:
@gforsyth I noticed that ion the same file we do have already `test_read_csv` and `test_read_csv_glob` similarly for parquet. Do we still want to convert these tests then? what are we testing that is different in this case?